### PR TITLE
chore(relayer): refactor logs and add comments in processor

### PR DESCRIPTION
**Pull request by xiaodino**
## Summary by CodeRabbit

- **New Features**
	- Added a new flag `crawl-past-blocks` to improve control over block processing.
- **Chores**
	- Updated `.gitignore` to ignore additional files.
- **Refactor**
	- Improved logging for transaction processing to enhance troubleshooting and monitoring.
	- Enhanced error handling in transaction processing for better stability and feedback.

### CODE REVIEW

File: packages/relayer/processor/process_single.go [hunk 1]
```diff
@@ -3,14 +3,18 @@ package processor
...
 	"strings"
 
 	"github.com/ethereum/go-ethereum/accounts/abi"
+	"github.com/ethereum/go-ethereum/common"
 	"github.com/taikoxyz/taiko-mono/packages/relayer/bindings/bridge"
 	"github.com/taikoxyz/taiko-mono/packages/relayer/pkg/queue"
 )
 
 func (p *Processor) processSingle(ctx context.Context) error {
+	slog.Info("Processing tx", "p.estimateGas:", common.Hash(*p.targetTxHash).Hex())
```
Comment (mask-pp) [inline packages/relayer/processor/process_single.go:16]:
```suggestion
	slog.Info("processing tx", "estimateGas", common.Hash(*p.targetTxHash).Hex())
```

File: packages/relayer/processor/process_single.go [hunk 2]
```diff
@@ -3,14 +3,18 @@ package processor
 import (
 	"context"
 	"encoding/json"
+	"log/slog"
 	"strings"
 
 	"github.com/ethereum/go-ethereum/accounts/abi"
+	"github.com/ethereum/go-ethereum/common"
```
Comment (mask-pp) [inline packages/relayer/processor/process_single.go:10]:
```suggestion
	"github.com/ethereum/go-ethereum/common"
	
```

File: packages/relayer/processor/processor.go [hunk 3]
```diff
@@ -411,13 +416,12 @@ func (p *Processor) eventLoop(ctx context.Context) {
...
 
 				if err != nil {
-					slog.Error("err processing message", "err", err.Error())
-
 					if errors.Is(err, errUnprocessable) {
 						if err := p.queue.Ack(ctx, msg); err != nil {
 							slog.Error("Err acking message", "err", err.Error())
 						}
 					} else {
+						slog.Error("err processing message", "err", err.Error())
```
Comment (mask-pp) [inline packages/relayer/processor/processor.go:424]:
```suggestion
						slog.Error("process message failed", "err", err.Error())
```

**Review by mask-pp — APPROVED** [review]
LGTM, just some carp.

**Review by davidtaikocha — APPROVED** [review]

**Pull request merged**